src/java/za/co/mip/ablduck/models/data/SubsectionDataObject.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2017 MIP Holdings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package za.co.mip.ablduck;
+
+import com.google.gson.annotations.SerializedName;
+import com.google.gson.annotations.Expose;
+
+
+public class SubsectionDataObject {
+
+	@Expose
+	String title = "";
+
+	@Expose
+	@SerializedName("default")
+	Boolean default_ = null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "default_" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS00116)